### ISSUE TITLE
Problem running the Scripts for preparing a class

### ISSUE THREAD

**Issue by EduSemensati** [githubtraining/training-manual#250]
**Team**, this is my first time trying to run this training and so far I got some good directions from @rwnfoo .
However, I am stuck in trying to run the scripts you folks provide in the repo.

I am using a Windows Machine, so I am trying to execute those through the Git Bash CLI. I installed jq (downloaded the executable and added to the path), but still facing the same issue (see below). I tried all the options in the menu, but script always fails when it tries to call another script during the execution.

![image](https://user-images.githubusercontent.com/24209506/91326826-0fed3980-e7c5-11ea-97a1-b3b885ea764e.png)

Then since I have Ubuntu installed in my WSL, I tried running things in there as well (after installing jq with apt-get) but got a different error which I must say I have no idea what actually means 😄 

![image](https://user-images.githubusercontent.com/24209506/91327127-6d818600-e7c5-11ea-913b-cb1d751ae9d6.png)

**Comment by parkerbxyz** [comment]
Hey @EduSemensati! 👋 

The error you're getting in Git Bash will occur if you run the script from within the `script` folder.

If you navigate to the root of the repository and then run `script/teach-class` it should work. 😊

**Comment by EduSemensati** [comment]
**So, first of all, thank you @parkerbxyz , @rwnfoo and @beardofedu for the AMAZING help and patience in going through that and doing the proper setup with me, HIGHLY APPRECIATED**, especially since I was not expecting to get support from GitHub on this 🥇 

Listing the problems here for future reference (most of them my fault, but as I said in the call, I will blame it on my long working hours today hehe):

1. **For the problem in this Issue:** I was trying to run the script from the script folder instead of the repository root (I misunderstood the documentation so when I saw script/teach-class, I assumed I needed to cd into that)
2. **Repo name is taken error:** the first time I ran the script with option 0 to do the setup, it had some issues and didnt complete successfully. I then edited the config file manually AND recreated it using the script. Nothing seemed to work until I decided to close the git bash CLI and open it up again, which apparently loaded the new config file and executed the script properly (YAY for the old support trick of just "turn it off and on again")
3. j**q not found:** If you are on Windows and download the .exe (instead of using Chocolatey), you will get a file named jq-win64 or similar. I had to: (1) add the folder where I stored the .exe to the PATH AND (2) rename the .exe to just jq.
4. **new-virtual line 52 error "No such file or directory":** it turns out that during all the troubleshooting, we edited the scope of the PAT which automatically removes the SSO Authorization. Since I missed that, the script was failing in the push step where it should create the repos. I re-authorized it and everything worked just fine!

![Alt Text](https://media1.tenor.com/images/8cb97d1ea5789f5cbb28cdc48a818435/tenor.gif?itemid=14528069)
